src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { findUserById } from '../../services/findUserById';
+import { Todo } from '../../types/todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={classNames('TodoInfo', {
+        'TodoInfo--completed': todo.completed,
+      })}
+    >
+      <h2 className="TodoInfo__title">
+        {todo.title}
+      </h2>
+      <UserInfo user={findUserById(todo.userId)} />
```

Comment:
user can be null, you need to handle this case here

Reply:
I decide to move this check using `user?.name` into `TodoInfo` component. In case `User` is `null` no info will be displayed,  but in current implementation we have no possibilities no meet `User === null`, because when it`s null we`ll recive ann error 'Please choose user'

Reply:
@Daniilart01  but you don't know where else this component can be reused so need to check if user exist